scripts/automated-patch-releases.md

```diff
@@ -0,0 +1,49 @@
+# CRI-O Automated Patch Releases
+
+Automated Patch Releases provides an easy way of packaging and release new version
+of CRI-O
+This involves the use of Golang Scripts and Github Actions
+
+```mermaid
+    flowchart TD
+      A[Start: patch-release.yml Workflow] --> B[Create version bump intent prs]
+      B --> C[Maintainers Review PRs]
+      C -->|Decides Against Merges| E[Rebases on main branch] --> C
```

Comment:
Mention the workflow as well as when (on the 1st of a month) the rebase happens.